### PR TITLE
grokj2k 9.7.1

### PR DESCRIPTION
**Pull request by cho-m**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cho-m** [comment]
Not sure why, but ARM RPATH is pointing to build directory `/tmp/...` and common `/usr/local/lib`, `/usr/lib`:
```
==> /opt/homebrew/Cellar/grokj2k/9.7.1/bin/grk_compress -InputFile Tless0.tiff -OutputFile test.jp2 -OutFor jp2 -TransferExifTags
  dyld[79275]: Library not loaded: @rpath/libgrokj2k.1.dylib
    Referenced from: /opt/homebrew/Cellar/grokj2k/9.7.1/libexec/bin/grk_compress
    Reason: tried: '/tmp/grokj2k-20220222-75451-kw97jl/grok-9.7.1/build/bin/libgrokj2k.1.dylib' (no such file), '/tmp/grokj2k-20220222-75451-kw97jl/grok-9.7.1/build/bin/libgrokj2k.1.dylib' (no such file), '/usr/local/lib/libgrokj2k.1.dylib' (no such file), '/usr/lib/libgrokj2k.1.dylib' (no such file)
  Error: grokj2k: failed
```

File: Formula/grokj2k.rb [hunk 1]
```diff
@@ -22,36 +22,128 @@ class Grokj2k < Formula
...
+  # Build failed with Apple clang version 12.0.5 and 13.0.0.
+  # clang: error: unable to execute command: Segmentation fault: 11
+  # clang: error: clang frontend command failed due to signal (use -v to see invocation)
+  # Upstream issue closed as Apple clang bug: https://github.com/GrokImageCompression/grok/issues/256
+  fails_with :clang if DevelopmentTools.clang_build_version >= 1205
+
+  resource "test_image" do
+    url "https://raw.githubusercontent.com/GrokImageCompression/input_image_test_suite/173de0ae73371751f857d16fdaf2c3301e54a3a6/exif-samples/tiff/Tless0.tiff"
+    sha256 "32f6aab90dc2d284a83040debe379e01333107b83a98c1aa2e6dabf56790b48a"
+  end
```
Comment (SMillerDev) [inline Formula/grokj2k.rb:59]: This should stop `bump` from suggesting updates for it.
```suggestion
  resource "homebrew-test_image" do
    url "https://raw.githubusercontent.com/GrokImageCompression/input_image_test_suite/173de0ae73371751f857d16fdaf2c3301e54a3a6/exif-samples/tiff/Tless0.tiff"
    sha256 "32f6aab90dc2d284a83040debe379e01333107b83a98c1aa2e6dabf56790b48a"
  end
```

File: Formula/grokj2k.rb [hunk 2]
```diff
@@ -22,36 +22,128 @@ class Grokj2k < Formula
...
-    system "cmake", ".", *std_cmake_args, "-DBUILD_DOC=ON"
-    system "make", "install"
+    args = ["-DBUILD_DOC=ON"]
+
+    if DevelopmentTools.clang_build_version >= 1205
+      ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
+      ENV.llvm_clang
+      # Workaround issue with LLVM install_name_tool, which causes errors during install.
+      # .../llvm/bin/install_name_tool: error: unsupported load command (cmd=0x80000034)
+      args << "-DCMAKE_INSTALL_NAME_TOOL=/usr/bin/install_name_tool"
```
Comment (cho-m) [inline Formula/grokj2k.rb:69]: @carlocab This workaround may be related to #95319. LLVM 13's `install_name_tool` is causing issues for ARM when modifying RPATH.

Comment (carlocab) [reply Formula/grokj2k.rb:69]: Yea, we should revert that part

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**